src/features/harborView/HarborViewContainer.tsx

```diff
@@ -15,6 +15,9 @@ import PierCreateForm from './forms/pierForm/PierCreateForm';
 import PierEditForm from './forms/pierForm/PierEditForm';
 import HarborEditForm from './forms/harborForm/HarborEditForm';
 import HarborViewTable from './HarborViewTable';
+import ContactInformationCard from '../../common/contactInformationCard/ContactInformationCard';
+import ActionHistoryCard from '../../common/actionHistoryCard/ActionHistoryCard';
+import styles from './harborView.module.scss';
```

Comment:
Importing the styles in two places might be problematic in the future, and usually, containers are style-free components.

Reply:
HarborView and HarborViewContainer were a bit of a mess, as HarborView did practically nothing and HarborViewContainer did a lot of rendering. I refactored them a bit and fixed this.

Reply:
Well done!